src/components/redemptions/ActivityCard.jsx

```diff
@@ -29,16 +30,25 @@ class ActivityCard extends Component {
   };
 
   static defaultProps = {
+    center: '',
     showUserDetails: false,
   };
-  statuses = ['pending', 'expired', 'approved', 'default'];
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      statuses: ['pending', 'rejected', 'approved', 'default'],
```

Comment:
This is Why I recommended a map above. You shouldn't have to repeat yourself.

Reply:
My bad, this is an unused component, should've been deleted. I'm reusing the ActivityCard in `components/activities`